deploy/overlays/dev-template/kustomization.yaml

```diff
@@ -46,7 +46,7 @@ patches:
          name: QUARKUS_S3_ENDPOINT_OVERRIDE
          value: "http://localstack.default.svc.cluster.local:4572"
    target:
-     kind: Task
+     kind: ClusterTask
```

Comment:
interesting

can you clarify the reason it make sense to go from namespace scope to cluster scope?

Reply:
we got into this one last night too ... this is in response to Michal changing the target over in infra / build defs as part of his resource limiting effort